float64 z_height
float64 rgbd_roll
float64 rgbd_pitch
float64 rgbd_yaw

